app/app/local.env

```diff
@@ -30,3 +30,6 @@ GITHUB_CLIENT_SECRET=
 GITHUB_APP_NAME=
 
 INFURA_USE_V3=True
+
+SENDGRID_API_KEY=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
```

Comment:
🤔 @thelostone-mc

Reply:
Oh derp ! 
Haha thanks man 🙌 you da best